examples/examples/ws_pubsub_broadcast.rs

```diff
@@ -116,20 +112,19 @@ async fn pipe_from_stream_with_bounded_buffer(
 				// and you might want to do something smarter if it's
 				// critical that "the most recent item" must be sent when it is produced.
 				if sink.send(notif).await.is_err() {
-					break;
+					break Ok(());
 				}
 
 				closed = c;
 			}
 
-			// stream is closed or some error, just quit.
-			Either::Right((_, _)) => {
-				break;
-			}
+			// Send back back the error.
+			Either::Right((Some(Err(e)), _)) => break Err(e.into()),
```

Comment:
It's nice that it's easy to return an error like this now!

Reply:
yeah, the only annoyance is if you want a JSON value here.

Then you would need to `serde_json::to_string(&v)` and the `Subscription::from(String)` would add an extra `""kakak""`.... but that's sacrificed with the StdError/ToString here :(